app/src/processing/app/Base.java

```diff
@@ -1513,18 +1513,14 @@ public void actionPerformed(ActionEvent actionevent) {
       boardsCustomMenus.add(customMenu);
     }
 
-    menuItemsToClickAfterStartup = new LinkedList<>();
+    List<JMenuItem> _menuItemsToClickAfterStartup = new LinkedList<>();
     boardsButtonGroup = new ButtonGroup();
     recentBoardsButtonGroup = new ButtonGroup();
     buttonGroupsMap = new HashMap<>();
 
-    if (BaseNoGui.getRecentlyUsedBoards() != null) {
-      JMenuItem recentLabel = new JMenuItem(tr("Recently used boards"));
-      recentLabel.setEnabled(false);
-      boardMenu.add(recentLabel);
-      rebuildRecentBoardsMenu();
-      //rebuildRecentBoardsMenu(null);
-    }
+    JMenuItem recentLabel = new JMenuItem(tr("Recently used boards"));
+    recentLabel.setEnabled(false);
+    boardMenu.add(recentLabel);
```

Comment:
This also seems like a change unrelated to the click after startup commit that is better squashed into the previous commit.